examples/README.md

```diff
@@ -1,49 +1,74 @@
 ![banner](https://github.com/iotaledger/identity.rs/raw/HEAD/.github/banner_identity.svg)
 
-# IOTA Identity Examples
+## IOTA Identity Examples
 
-This folder provides code examples to learn how IOTA Identity can be used.
+The following code examples demonstrate how to use IOTA Identity.
 
-You can run each example using
+### Prerequisites
 
-```rust
-cargo run --example <example_name>
-```
+Examples can be run against
+- a local IOTA node
+- or an existing network, e.g. the IOTA testnet
+
+When setting up the local node, you'll also need to publish an identity package as described in
+[Local Network Setup](https://docs.iota.org/iota-identity/getting-started/local-network-setup) in the documentation portal.
+You'll also need to provide an environment variable `IOTA_IDENTITY_PKG_ID` set to the package-id of your locally deployed
+identity package, to be able to run the examples against the local node.
+
+In case of running the examples against an existing network, this network needs to have a faucet to fund your accounts (the IOTA testnet (`https://api.testnet.iota.cafe`) supports this), and you need to specify this via `NETWORK_URL`.
+
+The examples require you to have the node you want to use in the iota clients "envs" (`iota client env`) configuration. If this node is configured as `localnet`, you don't have to provide it when running the examples, if not, provide its name as `NETWORK_NAME_FAUCET`. The table below assumes - in case you're running a local node - you have it configured as `localnet` in your IOTA clients "env" setting.
+
+### Environment variables
```

Comment:
The current variable setup for the Rust examples is a bit different from the TypeScript examples. Running against localnet works as usual, but to start a test against `testnet`, these must be provided:

- `API_ENDPOINT` - the same as `NETWORK_URL` in the TypeScript examples
- `IOTA_IDENTITY_PKG_ID` - currently always mandatory for the Rust examples

E.g.:

```bash
API_ENDPOINT=https://api.testnet.iota.cafe IOTA_IDENTITY_PKG_ID=0x222741bbdff74b42df48a7b4733185e9b24becb8ccfbafe8eac864ab4e4cc555 cargo run --release --example 0_create_did
```

Also the IOTA client must be configured to run against the same network, e.g. `iota client switch --env testnet` (if configured with that name).

Reply:
what about `NETWORK_NAME_FAUCET`?
Could you maybe make the appropriate changes on this PR?

Reply:
`NETWORK_NAME_FAUCET` is not used in the Rust tests, there we use the IOTA client's `faucet` subcommand to request funds. ^^